integreat_cms/cms/migrations/0035_alter_chatmessage_options.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2.15 on 2022-08-26 12:09
+
+from django.db import migrations
+
+from ..constants import roles
+
+# pylint: disable=unused-argument
+def update_roles(apps, schema_editor):
+    """
+    Update permissions for service and management group
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
```

Comment:
```suggestion

    # Emit post-migrate signal to make sure the Permission objects are created before they can be assigned
    emit_post_migrate_signal(2, False, "default")

```
Wow, this really confused me... I couldn't figure out why the migration wouldn't work for the `change_chatmessage` permission, apparently the migration changing the model options does not create the new permission, but the auth app waits for the migrations to finish and only after that it creates them... Unless we emit this signal manually beforehand.